java/src/main/native/src/row_conversion.cu

```diff
@@ -1652,20 +1746,18 @@ void determine_tiles(std::vector<size_type> const &column_sizes,
  * @param tbl table to convert to JCUDF row format
  * @param batch_info information about the batches of data
  * @param offset_functor functor that returns the starting offset of each row
- * @param column_starts starting offset of a column in a row
- * @param column_sizes size of each element in a column
- * @param fixed_width_size_per_row size of fixed-width data in a row of this table
+ * @param column_info information about incoming columns
+ * @param variable_width_offsets optional vector of offsets for variable-with columns
  * @param stream stream used
  * @param mr selected memory resource for returned data
  * @return vector of list columns containing byte columns of the JCUDF row data
  */
 template <typename offsetFunctor>
-std::vector<std::unique_ptr<column>>
-convert_to_rows(table_view const &tbl, batch_data &batch_info, offsetFunctor offset_functor,
-                std::vector<size_type> const &column_starts,
-                std::vector<size_type> const &column_sizes,
-                size_type const fixed_width_size_per_row, rmm::cuda_stream_view stream,
-                rmm::mr::device_memory_resource *mr) {
+std::vector<std::unique_ptr<column>> convert_to_rows(
+    table_view const &tbl, batch_data &batch_info, offsetFunctor offset_functor,
```

Comment:
Quick scan of the function makes me think this:
`batch_data &batch_info `
could be
`batch_data const& batch_info `

Reply:
I've tried this every time I look at it only to be reminded that we're releasing the cudf row offsets from inside this struct.